File: components/amp/test/test_spl06.js
/*
Please add this section into app.json when run this script as app.js.
This configuration is designed for haas100 edu k1.
"spl06": {
    "type": "I2C",
    "port": 1,
    "addrWidth": 7,
    "freq": 400000,
    "mode": "master",
    "devAddr": 119
}
*/

console.log('testing spl06...');

var spl06 = require('./spl06.js');

spl06.init("spl06");

var version = spl06.getID();
console.log("spl06 version is: " , version);

spl06.spl06_getdata();

spl06.deinit();
console.log("test spl06 success!");

File: components/py_engine/tests/haas/HaaS100/python-apps/driver/i2c/test_spl06.py
"""
Testing spl06 python driver

The below i2c configuration is needed in your board.json.
"spl06": {
  "type": "I2C",
  "port": 1,
  "addrWidth": 7,
  "freq": 400000,
  "mode": "master",
  "devAddr": 119
}
"""

from spl06 import SPL06

print("Testing spl06 ...")
spl06Dev = SPL06()
spl06Dev.open("spl06")
spl06Dev.init()
print("spl06 init finished")

id = spl06Dev.getID()
print("getID: " + str(id))

spl06Dev.spl06_getdata()

spl06Dev.close()

print("Test spl06 success!")
